src/Chapter2.hs

```diff
@@ -336,7 +336,10 @@ from it!
 ghci> :l src/Chapter2.hs
 -}
 subList :: Int -> Int -> [a] -> [a]
-subList = error "subList: Not implemented!"
+subList x y l
+  | x < 0 || y <0 = []
+  | x > y = []
+  | otherwise = drop x (take (y+1) l)
```

Comment:
It is recommended to add spaces on both sides of binary operators in Haskell :arrow_left: :space_invader: :arrow_right: 
```suggestion
  | otherwise = drop x (take (y + 1) l)
```